docs/cellxgene_census_FAQ.md

```diff
@@ -0,0 +1,109 @@
+- [Can I query human and mouse data in a single query?](#Can-I-query-human-and-mouse-data-in-a-single-query)
+- [Where are the Census data hosted?](#Where-are-the-Census-data-hosted)
+- [How can I increase the performance of my queries?](#How-can-I-increase-the-performance-of-my-queries)
+- [Can I use conda to install the Census Python API?](#Can-I-use-conda-to-install-the-Census-Python-API)
+- [How can I ask for support?](#How-can-I-ask-for-support)
+- [How can I ask for new features?](#How-can-I-ask-for-new-features)
+- [How can I contribute my data to the Census?](#How-can-I-contribute-my-data-to-the-Census)
+- [Why do I get an `ArraySchema` error when opening the Census?](#Why-do-I-get-an-ArraySchema-error-when-opening-the-Census)
+
+## Why should I use the Census?
+
+The Census provides efficient low-latency access via Python and R APIs to most single-cell RNA data from [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/). 
+
+To accelerate your computational research, **you should use the Census**:
+
+- If you want to easily get slices of data from more than 400 single-cell datasets spanning about 50 mi cells from >60 K genes.
+- If you want easy access to a subset of data from a dataset or from multiple datasets.
+- If you want to get these data with standardized and harmonized cell and gene metadata.
+- If you want to easily load multi-dataset slices into Scanpy or Seurat.
+- If you want to implement out-of-core (a.k.a online) operations for larger-than-memory processes.
+
+
+For example you could easily get "*all T-cells from Lung with COVID-19*" into an [AnnData](https://anndata.readthedocs.io/en/latest/), [Seurat](https://satijalab.org/seurat/), or into memory-sufficient data chunks via [PyArrow](https://arrow.apache.org/docs/python/index.html) or [R Arrow](https://arrow.apache.org/docs/r/). 
+
+
+**You should not use the Census:**
+
+- If you want access to non-standardized cell metadata and gene metadata available in the original [datasets](https://cellxgene.cziscience.com/datasets).
+- If you want access to the author-contributed normalized expression vaues.
+- If you want access to all data from a single dataset.
+
+## Why does the Census not have a normalized layer or embeddings?
+
+The Census does not have normalized counts or embeddings because:
+
+- The original normalized values and embeddings are not harmonized or integrated across datasets and therefore numerically incompatible.
+- We have not chosen nor implemented a general-purpose normalization or embedding generation method to be used across all Census data.
+
+If you have any suggestions for methods that our team should explore please share them with us via a [feature request in the github repository](https://github.com/chanzuckerberg/cellxgene-census/issues/new?assignees=&labels=user+request&template=feature-request.md&title=).
+
+## How does the Census differentiate from other services?
```

Comment:
Why not replace **Why should I use the Census?** with this section. It basically is stating the same points.

Reply:
A little different, Why should I use the Census goes also into author meatadata, raw counts, embeddings which are available at Discover.

This one here is motivated to compare outside the whole product.

Reply:
Except it repeats/restates the earlier bullets.

Reply:
I think some level of repetition is ok in a FAQ